scripts/install/usr/libexec/cortx-motr/motr-free-space-monitor

```diff
@@ -123,9 +123,16 @@ def print_iem(usage_percentage):
     evt_id = iem_event_id
     msg = f"Cluster size {usage_percentage}"
     msg_b = msg.encode('utf-8')
-    libmotr.m0_time_init()
-    libmotr.m0_iem(ctypes.c_char_p(file_parm_b), function_parm, line_num,
-                        sev_id, mod_id, evt_id, ctypes.c_char_p(msg_b))
+
+#   Current libmotr api "m0_iem()" is generalize for all IEM alerts from motr.
+#   Which is not allowing to provide free space monitor alert message as
+#   required format, hence added print log to provide alert in required format.
+#   Keeping exiting logic as well (commented) as it may required in LDR-R2
+
+    log.error(f"IEC: AS0020020002: Usable storage is {usage_percentage} % full");
```

Comment:
Do you mean, the log will be parsed by IEM and then be shown on GUI?

Reply:
Yes. It will parse though journal logs for IEM logs and will be shown to GUI.